meteor-app/imports/ui/Report_Builder/verifyReport.ts

```diff
@@ -0,0 +1,59 @@
+import { ReportStructure } from '../../api/types/reports'
+
+
+// {_id : Match.Maybe(String), account_id : Match.Maybe(String), name : String, 
+// 	tables : [{id : String, title : String, type : String, 
+// 		columns : [{id : String, label : String, formulaId : Match.Maybe(String), property : Match.Maybe(String), collection_name : String, relation_key : Match.Maybe(String), enum : String, symbol : Match.Maybe(String)}], 
+// 		rows : [{id : String, 
+// 			cells : [{id : String, index : Number, type : String, property : String, propertyValue : String, value : String, expression : String}] }], 
+// 		collection : String, sort_by : String}], 
+// 	formulas : [{id : String, tableId : String, columnId : String, columnIndex : Number, expression : String, 
+// 		// some of the formula types were decided by looking at how they are created in the columnToolBar
+// 		values : [{key : String, type : String, operation : String, collection_name : Match.Maybe(String), queryModifier : String, query : {collection_name : String}, property : Match.Maybe(String), path : Match.Maybe(String), columnId : Match.Maybe(String), cellIndex : Match.Maybe(String)}] }], 
+// 	public : Boolean, tags : [String] }
+
+export const verifyReport = (report : ReportStructure) => {
+	
+	if (!report) return 'No report found'
+	
+	if (!report.name) return 'Please name report'
+
+	if (report.tables.length > 0) {
+		
+		for (const table of report.tables) {
+			
+			if (!table.title) return 'Please name table'
+			
+			if (table.type === 'collection' && !table.collection) return 'Please select collection for ' + table.title + ' table'
+			
+			if (table.columns.length > 0) {
```

Comment:
Lets add a check to make sure there is at least 1 column in the report within a given table